ActionHandler/ZgwToVrijbrpRelocationHandler.php

```diff
@@ -50,13 +50,13 @@ public function getConfiguration(): array
                     'required'    => true,
                     '$ref'        => 'https://commongroundgateway.nl/commongroundgateway.gateway.entity.json',
                 ],
-                'foundBodyLocation' => [
+                'interLocation' => [
                     'type'        => 'string',
                     'description' => 'The endpoint we will use on the Source to send a request, just a string',
```

Comment:
Kloppen de descriptions nog?

Reply:
Yes, functie van de velden veranderd niet, alleen is dit ooit foutief gekopiëerd